mne/report.py

```diff
@@ -769,6 +769,9 @@ def _add_figs_to_section(self, figs, captions, section='custom',
             warnings.warn('Could not import mayavi. Trying to render '
                           '`mayavi.core.scene.Scene` figure instances'
                           ' will throw an error.')
+        if np.isscalar(scale) and scale < 1.:
+            raise ValueError('scale, if float, must be >= 1 not %s'
+                             % scale)
```

Comment:
The discussion here indicates that `scale >= 1` is the only safe condition:

https://github.com/mne-tools/mne-python/issues/2014#issue-70542876

If you have a better solution I'm all ears...